.cirrus.yml

```diff
@@ -80,7 +80,7 @@ optional_system_testing_task:
 
     gce_instance:
         matrix:
-            image_name: "ubuntu-1804-bionic-v20180911-libpod-63a86a18"
+            image_name: "ubuntu-18-libpod-0c954a67"
```

Comment:
FYI- this image was built with ``RUNC_COMMIT`` 4932620b6237ed2a91aa5b5ca8cca6a73c10311b

Reply:
ah thanks! How we can re-build it with the correct commit?  I think this is the reason of the failures